sonar-project.properties

```diff
@@ -0,0 +1,13 @@
+sonar.projectKey=Om-Sevak_ClubHub
+sonar.organization=om-sevak
+
+# This is the name and version displayed in the SonarCloud UI.
+#sonar.projectName=ClubHub
+#sonar.projectVersion=1.0
+
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+#sonar.sources=.
```

Comment:
Please exclude files in server/test. See: https://docs.sonarsource.com/sonarcloud/advanced-setup/analysis-scope/ for how to.